packages/shared/world/parcelSceneManager.ts

```diff
@@ -122,99 +145,260 @@ function reportPendingScenes() {
   store.dispatch(informPendingScenes(pendingScenes.size, countableScenes))
 }
 
-export async function enableParcelSceneLoading(options: EnableParcelSceneLoadingOptions) {
-  const ret = await initParcelSceneWorker()
+const parcelSceneLoadingState = {
```

Comment:
Added the type for the parcelSceneLoadingState, for the payload, it can be different for each mode, right now there is only the builder mode but in the future, it can have different values, we still want to type it?